app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v3.rb

```diff
@@ -110,17 +110,17 @@ def powered_on_in_provider?(vm)
     end
 
     def vm_boot_from_cdrom(operation, name)
-      operation.get_provider_destination.boot_from_cdrom(name)
+      operation.with_provider_destination { |d| d.boot_from_cdrom(name) }
     rescue Ovirt::VmNotReadyToBoot
       raise OvirtServices::VmNotReadyToBoot
     end
 
     def detach_floppy(operation)
-      operation.get_provider_destination.detach_floppy
+      operation.with_provider_destination { |d| d.detach_floppy }
```

Comment:
I think rubocop suggests you use:  operation.with_provider_destination(&:detach_floppy)